magento/ece-tools
# MCLOUD-5325: Move log section from .magento.env.yaml

### Description
https://jira.corp.magento.com/browse/MCLOUD-5325

### Manual testing scenarios

1. Clone magento-cloud project locally
2. Run command `vendor/bin/ece-tools schema:generate`
3. Check that the generated file contains content from `vendor/magento/ece-tools/dist/.log.env.md` at the end of the file.

### Contribution checklist
